src/controller/CHIPDeviceController.cpp

```diff
@@ -2565,7 +2613,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         // This is done in a separate step since we've already used up all the available read paths in the previous read step
         // NOTE: this array cannot have more than 9 entries, since the spec mandates that server only needs to support 9
         // See R1.1, 2.11.2 Interaction Model Limits
-        app::AttributePathParams readPaths[3];
+        app::AttributePathParams readPaths[5];
```

Comment:
Could we have a define somewhere instead of this magic number? We want from 3 to 5 and the comment above states the max is 9. A define somewhere with this comment attached to it would be helpful for clarity.

Reply:
I think 5 here is not a magic number, since we have exactly 5 attributes to read below.
Added a comment above should be enough.